src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.ts

```diff
@@ -1,24 +1,40 @@
-import {Component, model, OnInit} from '@angular/core';
+import {Component, input, model, OnInit, output} from '@angular/core';
+import {
+  NavigatorComponentGrouping,
+  NavigatorComponentSorting
+} from '../complex-navigator/complex-navigator-utils';
+import {ComplexSearchResult} from '../../shared/model/complex-results/complex-search.model';
 
 @Component({
   selector: 'cp-complex-navigator-buttons',
   templateUrl: './complex-navigator-buttons.component.html',
   styleUrls: ['./complex-navigator-buttons.component.css']
 })
 export class ComplexNavigatorButtonsComponent implements OnInit {
-  interactorsSorting = model<string>();
+  componentsSorting = model<NavigatorComponentSorting>();
```

Comment:
We don't need this anymore, as we have the output. You can delete this, delete `this.componentsSorting.set(typeOfSorting);` on line 37 and delete `[(componentsSorting)]="componentsSorting"` in complex-navigator.component.html (line 4).